### ISSUE TITLE
[discussion] Having an array of videos instead of a single video_url

### ISSUE THREAD

**Issue by EddyVinck** [EddyVinck/ConfTalks#9]
As mentioned here: https://github.com/EddyVinck/ConfTalks/pull/5#discussion_r336760839

I'm open to ideas / suggestions

**Comment by EddyVinck** [comment]
I just had the idea to add a detail page for talks. So there would be a page for every talk like conftalks.org/talk/<id> where you would find more info about the talk. This way there would only need to be one more additional button on the regular talk card like "More info".

**Comment by taksuparth** [comment]
In more info page we could show different versions of that talk if we keep `video_url` an array. On the regular talk card, we can keep the watch URL as the first item of `video_url` array. This way main page won't be so cluttered with the same talk plus, in future if we want there we can show related videos too. Videos which belong to that same topic or videos which are of the same conference.

**Comment by EddyVinck** [comment]
@taksuparth I agree that it should be implemented like that. Related videos on the detail page is a good idea as well! :+1:
